pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TopicStats.java

```diff
@@ -42,6 +42,9 @@ class TopicStats {
     long bytesOutCounter;
     double averageMsgSize;
 
+    long ongoingTxnCount;
```

Comment:
because topic transaction stats need transaction buffer stats and pending ack stats, so I think it only represent tb ongoingTxnCount